network/local.go

```diff
@@ -176,7 +179,14 @@ func (lm *LocalManager) count() int {
 // Stop tells any connections that are sleeping on retry
 // to stop sleeping and return an error.
 func (lm *LocalManager) Stop() {
-	lm.stopOnce.Do(func() { close(lm.stopping) })
+	lm.Lock()
```

Comment:
stopOnce was needed before due to panic on "closing a closed channel" for the close of lm.stopping. What changed to not need this?

Reply:
It seems that we call `Stop` only once now. What test was triggering the error?

Reply:
Don't recall. Let's check this in and come back to it if we need to.